pkg/util/utils.go

```diff
@@ -59,6 +62,11 @@ func StringMapToCommaString(labels map[string]string) string {
 	return strings.Join(s, ",")
 }
 
+func GetRandom() uint64 {
```

Comment:
Probably should see if we can use something similar to genVolName() in common_test. Probably create a 
**func GetRandomName(prefix string) string** 
 in util.go and call that in genVolName as well as anywhere else we need this.

Reply:
Fixed by adding GetRandomName() instead of GetRandom. Also modified the genVolName to use it.